RadixWallet/Clients/ImportLegacyWalletClient/ImportLegacyWalletClient+Live.swift

```diff
@@ -41,10 +41,10 @@ extension ImportLegacyWalletClient: DependencyKey {
 
 				let appearanceID = await accountsClient.nextAppearanceID(networkID, accountOffset)
 
-				let babylon = try Account(
+				let babylon = Account(
```

Comment:
many previously throwing methods and inits are no longer throwing in Sargon. We should all help out cascading those changes to calling scopes (and scopes calling those calling scopes...)